.github/workflows/check-code-style.yml

```diff
@@ -24,7 +24,7 @@ env:
   # We need compile command database in order to perform clang-tidy check. So,
   # in order to perform configure step we need to setup llvm-dev package. This
   # env variable used to specify desired version of it
-  LLVM_VERSION: 14
+  LLVM_VERSION: 13
```

Comment:
A similar change is needed for `.travis.yml`

Reply:
Should we remove `.travis.yml`? It seems nobody needs binaries anyway.

Reply:
No objections personally, but we should probably decide that elsewhere for visibility (perhaps tooling call?).

While waiting for a decision, I'm okay to leave Travis CI broken/disabled, so that we can proceed with the llvm_release_130 branch.

Reply:
Agree about the visibility.